paper.tex

```diff
@@ -588,6 +588,9 @@ \subsection{Sizes and epochs}
 the existence interval of the population. Each epoch specifies the population size
 over that interval, which can be a constant value or a function defined by start
 and end sizes that must remain positive.
+Only exponential population size changes are currently supported,
+but other functions may be added to the specification over time.
```

Comment:
I'm not sure this is completely true. The spec currently says:
https://popsim-consortium.github.io/demes-spec-docs/main/specification.html#size-function

So we say that "exponential" and "constant" population size changes should be interpreted in a certain way and don't exclude other possiblities. This was carefully worded because demes-python also recognises "linear", which is used by GADMA and can be plotted with demesdraw.

Reply:
I'm also supporting linear in the rust parser.

Reply:
Yeah, ok, this does keep the discussion simple.